src/Mod/Path/PathScripts/PathOpTools.py

```diff
@@ -195,8 +195,12 @@ def offsetWire(wire, base, offset, forward):
     if wire.isClosed():
         if not base.isInside(owire.Edges[0].Vertexes[0].Point, offset/2, True):
             PathLog.track('closed - outside')
+            if Side:
+                Side[0] = "Outside"
             return orientWire(owire, forward)
         PathLog.track('closed - inside')
+        if Side:
+            Side[0] = "Inside"
```

Comment:
I'm not sure I understand what's going on here. We let the user set on which side they want to deburr and then the operation overwrites that?

Reply:
Basically yes, because:
- this function has several paths, but only this one sets the side. Therefor i'm not sure if there is a case you have to set it manually
- You could hide the attribute from the user, but i don't know how

Reply:
>We let the user set on which side they want to deburr

To be more clear, the user should just ignore the side attribute since it is automatically set by the op. But in case of an error etc. there is still the possibility to set it manually

Reply:
The way this works though is the op replaces whatever the user sets. An if the op overwrites it anyway, why have a property for it?

Reply:
I'm still confused about the status of this PR.  Please address the reviewers comment.

Reply:
Hello,
see comment below (23 days ago).
I hid the Side-Property and now it is exclusivly set by the script.
I also merged the current master and changed a setup property.